tpkg/concurrent_credns.tdir/concurrent_credns.post

```diff
@@ -0,0 +1,11 @@
+# #-- concurrent_credns.post --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# source the test var file when it's there
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+#
+# do your teardown here
+
+make realclean
```

Comment:
Missing is `make stop` to stop the started NSD instances

Reply:
I guess that goes for `tpkg/credns.tdir/credns.post` to then. I'll add it.

Reply:
Not the case for the `credns` test. There it's located in `credns.test` near the bottom, but I think I'll move that one to the `post` script too. Better to stop the servers whether the test fails or succeeds.

Reply:
Done